src/components/Input/index.tsx

```diff
@@ -0,0 +1,58 @@
+import React, { useState } from 'react';
+import { ReactComponent as EyeOpen } from 'assets/svg/EyeOpen.svg';
+import './styles.scss';
+
+export type InputType = {
+  value: string | number;
+  label: string;
+  type: string;
+  name: string;
+  error?: string;
+  max?: number;
+  min?: number;
+  disabled?: boolean;
+  showOnInput?: boolean;
```

Comment:
That's if the type is password and you want to toggle the filed to show the password in text, this will handle it..

Though another solution to this is to the type (text or password)